components/ldap.rst

```diff
@@ -0,0 +1,66 @@
+.. index::
+   single: Ldap
+   single: Components; Ldap
+
+The Ldap Component
+==================
+
+    The Ldap component provides a means to connect to an Ldap server (OpenLdap or Active Directory).
```

Comment:
This description should be very concise, so you did it great. However, if this component adds LDAP support to the Symfony Security component, maybe we should mention it. Right now it looks like this component is stand-alone and not integrated with Symfony security.